packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -1252,7 +1271,12 @@ public void onSelectionChanged(int start, int end) {
       if (mPreviousSelectionStart != realStart || mPreviousSelectionEnd != realEnd) {
         mEventDispatcher.dispatchEvent(
             new ReactTextInputSelectionEvent(
-                mSurfaceId, mReactEditText.getId(), realStart, realEnd));
+                mSurfaceId,
+                mReactEditText.getId(),
+                realStart, 
```

Comment:
Another minor thing, but there's an extraneous space at the end of line 1276 and 1277

Reply:
👍🏻